packages/amplify-provider-awscloudformation/src/utils/delete-from-service.ts

```diff
@@ -8,7 +8,12 @@ import { getSsmSdkParametersDeleteParameters, getSsmSdkParametersGetParametersBy
  * Delete CloudFormation parameters from the service
  */
 export const deleteEnvironmentParametersFromService = async (context: $TSContext, envName: string): Promise<void> => {
-  const appId = resolveAppId(context);
+  let appId;
+  try {
+    appId = resolveAppId(context);
+  } catch {
+    return;
+  }
   const { client } = await SSM.getInstance(context);
   await deleteParametersFromParameterStore(appId, envName, client);
```

Comment:
```suggestion
  try {
    const appId = resolveAppId(context);
    const { client } = await SSM.getInstance(context);
    await deleteParametersFromParameterStore(appId, envName, client);
  } catch {
    return;
  }
  
```

Reply:
Is suppressing the error the desired behavior here?

Reply:
`resolveAppId` will throw if no appId can be determined, but we do not want to prevent delete from succeeding because not having an appId is a valid state (such as running `amplify pull` but not modifying the backend).